code/modules/antagonists/nukeop/equipment/nuclearbomb.dm

```diff
@@ -1,6 +1,6 @@
 /obj/machinery/nuclearbomb
 	name = "nuclear fission explosive"
-	desc = "You probably shouldn't stick around to see if this is armed."
+	desc = "A NanoTrasen branded nuclear bomb, You probably shouldn't stick around to see if this is armed."
```

Comment:
Specify that it's a stolen bomb and use a period instead of a comma because grammar

Reply:
ok